.github/workflows/update-test-images.yml

```diff
@@ -0,0 +1,23 @@
+name: Update Test Images
+on:
+  push:
+    branches: ['terraform']
+    
+    # List of paths that should be considered when triggering the workflow
+    paths: ['mocked_servers/**']
```

Comment:
Should we also publish for `sample_apps`also since `imagebuild` module does both?

Reply:
yes, I updated the list of watched directories.